em_workflows/test-runner/flow.py

```diff
@@ -27,12 +30,32 @@ def publish_artifact(report: str) -> None:
 
 
 @task
-def run_tests() -> str:
+def run_tests(git_branch: str) -> str:
     """
+    ***Limit to dev server***
     Run pytest and save coverage
     The returned string can be viewed in pytest server artifacts section
     Currently, https://prefect2.hedwig-workflow-api.niaiddev.net/artifacts/key/pytest-cov-report
     """
+    test_path = "test-runner"
+    test_dir = Path.home() / test_path
+    assert test_dir.exists(), "Make sure test-runner dir exists at HOME"
```

Comment:
what about
```
if not test_dir.exists():
   mkdir(test_dir)
```
?

Reply:
test-runner is our duplicate project repo in the dev machine. So this will be created one time, and exist there.

Reply:
Not applicable - ignoring